Cilsil/Program.cs

```diff
@@ -100,7 +101,14 @@ private static void Main(string[] args)
 
             (var cfg, var tenv) = ExecuteTranslation(paths, extprogress);
 
-            File.WriteAllText(cfgtxt ?? "./cfg.txt", cfg.ToString());
+            var Utf8Encoder = Encoding.GetEncoding(
+                "us-ascii",
+                new EncoderReplacementFallback("_"),
+                new DecoderExceptionFallback()
```

Comment:
Decode exception fall back string is missing here. You may consider use one that can be differentiated from encoder replace fall back string.
![image](https://user-images.githubusercontent.com/63893533/174414626-4ff5be65-5919-4cec-a7a9-21ad662e1c1f.png)

Reply:
done